modules/apps/knowledge-base/knowledge-base-web/src/main/java/com/liferay/knowledge/base/web/internal/display/context/KBAdminNavigationDisplayContext.java

```diff
@@ -405,6 +410,37 @@ private JSONArray _getKBArticleNavigationJSONArray(long parentFolderId)
 		return articleNavigationJSONArray;
 	}
 
+	private JSONArray _getTemplatesJSONArray() {
```

Comment:
My only doubt is if call this `_getTemplateNavigationJSONArray` because the method for the article is `_getKBArticleNavigationJSONArray`

Reply:
Makes sense, I will change it.